ctapipe/ml/tools/apply_energy_regressor.py

```diff
@@ -92,7 +92,7 @@ def start(self):
             table = self.loader.read_telescope_events([tel_id])
             table.remove_columns([c for c in table.colnames if c.startswith(prefix)])
 
-            if len(table) == 0:
+            if len(table) == 0 or "hillas_intensity" not in table.colnames:
```

Comment:
I guess this was for testing purposes? Or do you want to assume that column will always be there?

Reply:
It is connected to the now fixed issue with impact positions, I used this to check for the empty tables.